oci_python_image/MODULE.bazel

```diff
@@ -25,13 +32,17 @@ use_repo(pip, "pip")
 
 oci = use_extension("@rules_oci//oci:extensions.bzl", "oci")
 oci.pull(
-    name = "distroless_python",
-    # tag = "latest", # as of 30 August 2023
-    digest = "sha256:5148968d8ae02a0f6d12efaca7a16e711ab43a4695a285e22dbbae70d6048937",
+    name = "python_base",
```

Comment:
Distroless doesn't work?

Reply:
Unfortunately no, we depend on some bash features that are not available on sh.